configureWorkspace/configure.ts

```diff
@@ -94,6 +127,17 @@ services:
     ports:
       - ${port}:${port}`;
 
+        case 'python':
+            return `version: '2.1'
+
+services:
+${serviceName}:
```

Comment:
Improper indentation in YAML here and in the following lines.

Reply:
PR to fix at https://github.com/Microsoft/vscode-docker/pull/242